### DISCUSSION TITLE
Tilling option from AUTOMATIC to create seamless patterns

### DISCUSSION THREAD

**Discussion by C00reNUT** [vladmandic/sdnext#2740] [Q&A] [answered]
Hello may I ask you is there a Tilling option from AUTOMATIC to create seamless patterns? I cannot find it in GUI and Settings, it is present in AUTOMATIC as 
<img width="87" alt="image" src="https://github.com/vladmandic/automatic/assets/34869909/1e6fe401-441c-4550-a2e7-4a61634c4f3d">

**Comment by brknsoul** [comment, marked as answer]
Under the Advanced accordian;
![image](https://github.com/vladmandic/automatic/assets/1969381/89ff1f1c-1114-4d38-b402-7d5079f73ae4)

**Comment by C00reNUT** [comment]
> Under the Advanced accordian; ![image](https://private-user-images.githubusercontent.com/1969381/298551510-89ff1f1c-1114-4d38-b402-7d5079f73ae4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU5MjIyMDEsIm5iZiI6MTcwNTkyMTkwMSwicGF0aCI6Ii8xOTY5MzgxLzI5ODU1MTUxMC04OWZmMWYxYy0xMTE0LTRkMzgtYjQwMi03ZDUwNzlmNzNhZTQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDEyMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMjJUMTExMTQxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjA5ZmU2MzBhOWRkNjE5ZDc4MWJmMjUxNzZkOTYzMDY3MjFlOGRmZWIzZTI2NDdkYjFkZWVhMTAwZmIzZmU2OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.jianNewI08ftzbBAVfH3KbQHhM_WR1rozQzsFwqnVho)

Thank you! Sometimes I am blind :)

**Comment by C00reNUT** [comment]
<img width="519" alt="image" src="https://github.com/vladmandic/automatic/assets/34869909/9de22c30-10e2-4737-abb6-3c515a2bcc4d">
Actually I was trying to find it and couldn't. I know it was removed in AUTOMATIC in recent versions, but in that interface it is still present in Setting...

**Reply by vladmandic** [reply]
tiling is not implemented for backend diffusers

**Reply by C00reNUT** [reply]
Ou, I get it, I will use Automatic when I need it then, thank you for this wonderful UI tool!
